pkgs/by-name/li/libpciaccess/package.nix

```diff
@@ -0,0 +1,82 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  testers,
+  writeScript,
+  pkg-config,
+  meson,
+  ninja,
+  zlib,
+  netbsd,
+  hwdata,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libpciaccess";
+  version = "0.18.1";
+
+  src = fetchurl {
+    url = "mirror://xorg/individual/lib/libpciaccess-${finalAttrs.version}.tar.xz";
+    hash = "sha256-SvQ0RLOK21VF0O0cLORtlgjMR7McI4f8UYFlZ2Wm+nY=";
+  };
+
+  # not sure if needed
+  hardeningDisable = [
+    "bindnow"
+    "relro"
+  ];
```

Comment:
Where did these come from? Were they previously set for the whole package set?

Reply:
Yes all xorg packages have hardeningDisable = ["bindnow" "relro"].
something about dynamic shared object loading

from the nixpkgs manual:
> This flag can break dynamic shared object loading. For instance, the module systems of Xorg and PHP are incompatible with this flag.

i don't know how to test if it is actually needed for each package

Reply:
@risicle do you know?

Reply:
@thefloweringash viewing the git blame of `generate-expr-from-tarballs.pl`, i see that `hardeningDisable = ["bindnow" "relrow"];` comes from you through https://github.com/NixOS/nixpkgs/commit/9454f71fefcbe413fdfcb61b510256f1301c5f5c (#45328)

What has hardening to do with overrideScope'?
Why was it added to every xorg package?
Can it safely be removed?

Reply:
oh, i didn't see that
that is even longer ago

@globin can you contribute anything?
Do you remember in which way it broke the module system?
Or do you have any idea on how to find out which packages still need that?

Reply:
gentoo supposedly has bindnow and relro by default and does not seem to disable them for xorg packages

i didn't find the code that enables it tho, so im not sure